src/classes/HH_Container_LCTRL.cls

```diff
@@ -115,6 +142,7 @@ public class HH_Container_LCTRL {
             if (hhId == null)
                 return null;
 
+			// conSelect does the FLS check
```

Comment:
I find it very confusing to use property getters this way.  You're using the conSelect property like a function call.. a function call that takes implicit arguments from global scope, rather than being passed in, even.

I urge you to consider changing the conSelect property to a method call with a descriptive name.  Maybe even something like `getContactSelectQueryWithFls()`?